README.md

```diff
@@ -41,6 +41,38 @@ app locally. Do this like so:
     cd yari
     yarn install
 
+<details>
```

Comment:
Can you move this into the `## Troubleshooting` section further below?

Reply:
Good point!